<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Zagad2</class>
 <widget class="QWidget" name="Zagad2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>420</y>
     <width>350</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>Царь Эдип</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>550</y>
     <width>350</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>Человек</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>680</y>
     <width>350</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>Сфинкс</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>631</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;style&gt; p {text-indent: 50px} &lt;/style&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Вы решили повернуть направо. Пройдя несколько&lt;br/&gt;метров, вы натыкаетесь на дверь. Как только вы взялись&lt;br/&gt;за ее ручку, прозвучал голос: &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;&amp;quot;Раз ты смог пройти мою первую загадку то значит и&lt;br/&gt;с этой у тебя не возникнет трудов. На сей раз это&lt;br/&gt;последняя загадка. Правила ты уже знаешь, а вот и сама&lt;br/&gt;загадка. Кто утром ходит на двух ногах, днём на четырёх,&lt;br/&gt;а вечером на трёх?&amp;quot;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pict">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../PICT/pict2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>251</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>label_pict</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>label_2</zorder>
  <zorder>label5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
